packages/discovery-react-components/src/components/DocumentPreview/components/PdfViewer/PdfViewer.tsx

```diff
@@ -216,20 +238,28 @@ type CanvasInfo = {
   height: number;
   canvasWidth: number;
   canvasHeight: number;
+  proportion: number;
   viewport: PdfjsLib.PDFPageViewport;
 };
 
 function getCanvasInfo(
   loadedPage: PdfjsLib.PDFPageProxy | null | undefined,
-  scale: number
+  scale: number,
+  rootRef: RefObject<HTMLDivElement>
 ): CanvasInfo | null {
-  const canvasScale = window.devicePixelRatio ?? 1;
-  const viewport = loadedPage?.getViewport({ scale: scale * canvasScale });
-  if (viewport) {
+  const rootDimensions = rootRef?.current?.getBoundingClientRect();
+  if (loadedPage && rootDimensions) {
+    const canvasScale = window.devicePixelRatio ?? 1;
+    const width = rootDimensions.width * scale;
```

Comment:
Let's pick some better names here:
```suggestion
    const scaledWidth = rootDimensions.width * scale;
```

Reply:
Do you want the values returned by this function to be renamed accordingly as well (line 262)?

`return { width, height, canvasWidth, canvasHeight, proportion, viewport };`

Reply:
Well, just looking at that line, it is not evident to me how `width` and `canvasWidth` are different or what they represent. Also, what does `proportion` stand for?

Choosing more specific names may help, or maybe we can only handle these definitions through comments. Or both!

Reply:
Yeah, I mentioned to you that I'd need help naming `proportion` something more descriptive. I'll go ahead and push my changes that include comments and see if things a re more clear...